csrc/u8x8_d_st7565.c

```diff
@@ -143,7 +143,7 @@ uint8_t u8x8_d_st7565_common(u8x8_t *u8x8, uint8_t msg, uint8_t arg_int, void *a
 	controller: It is not allowed to write beyond the display limits.
 	This is in fact an issue within flip mode.
       */
-      if ( c + x > 132u )
+      if ( ((uint16_t) (c + x)) > 132u )
```

Comment:
As far as I understand (according to [this](http://stackoverflow.com/a/17882460)), `c + x` is promoted to `signed int` (because all the values of `uint8_t` can be expressed as `signed int`). Therefore, the warning is right, because `132u` is of type `unsigned int`. So to silence the warning, it should be casted to `unsigned int`, or remove the `u` modifier from 132.

I still don't understand why I cannot reproduce this with GCC.